integreat_cms/cms/views/pages/page_form_view.py

```diff
@@ -89,103 +88,27 @@ def get(self, request: HttpRequest, *args: Any, **kwargs: Any) -> HttpResponse:
-                            "region_slug": region.slug,
-                            "language_slug": language.slug,
-                            "page_id": page.id,
-                        },
-                    )
-                    if page_translation.status == status.AUTO_SAVE:
-                        status_message = _("The last changes were saved automatically.")
-                        action = _("discard")
-                    else:
-                        status_message = _("The latest changes are pending approval.")
-                        action = _("reject")
-                    # If the user has the permission to reject/discard, show another message
-                    if request.user.has_perm("cms.publish_page_object", page):
-                        message = __(
-                            status_message,
-                            _(
-                                "You can {action} these changes in the <a>version overview</a>.",
-                            ).format(action=action),
-                        )
-                        status_message = translate_link(
-                            message,
-                            attributes={
-                                "href": revision_url,
-                                "class": "underline hover:no-underline",
-                            },
-                        )
-                    messages.warning(request, status_message)
-                # Show information if a public translation exists even if the latest version is not public
-                if public_translation := page.get_public_translation(language.slug):
-                    if page_translation.status == status.DRAFT:
-                        messages.warning(
-                            request,
-                            _("The latest changes have only been saved as a draft."),
-                        )
-                    revision_url = reverse(
-                        "page_versions",
-                        kwargs={
-                            "region_slug": region.slug,
-                            "language_slug": language.slug,
-                            "page_id": page.id,
-                            "selected_version": public_translation.version,
-                        },
-                    )
-                    message = _(
-                        "Currently, <a>version {}</a> of this page is displayed in the app.",
-                    ).format(public_translation.version)
-                    messages.info(
-                        request,
-                        translate_link(
-                            message,
-                            attributes={
-                                "href": revision_url,
-                                "class": "underline hover:no-underline",
-                            },
-                        ),
-                    )
+        disabled = self.get_initial_disable_state(request, page)
```

Comment:
The value that is returned here is never used. If this is just to handle warning messages if the page is archived, I'd prefer renaming the method to something more fitting